src/main/java/net/sf/jabref/MetaData.java

```diff
@@ -455,4 +457,20 @@ public Charset getEncoding() {
     public void setEncoding(Charset encoding) {
         this.encoding = Objects.requireNonNull(encoding);
     }
+
+    public void clearMetaData() {
+        metaData.clear();
+    }
+
+    public Map<String, List<String>> getMetaData() {
```

Comment:
I would like to use another solution, but unfortunately there is no other. In my opinion there should exist appropriate classes and data structures for meta data which are not present (and my time capacity is exceeded to do that). The only thing I could do is to use `getAsStringMap` and `parse` but as I described above this would cause very uggly code (parsing the strings and compiling them to match the remote data structure).